proto/cmp/types/v1alpha/travel_type.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package cmp.types.v1alpha;
+
+enum TravelType {
+  TRAVEL_TYPE_UNSPECIFIED = 0;
+  TRAVEL_TYPE_TOURISM = 1;
+  TRAVEL_TYPE_BUSINESS = 2;
+}
```

Comment:
Looking at the OTA codelist, I suggest:

enum TravelType {
  TRAVEL_TYPE_UNSPECIFIED = 0;
  TRAVEL_TYPE_LEISURE = 1;
  TRAVEL_TYPE_BUSINESS = 2;
  TRAVEL_TYPE_CONFERENCE = 3;
  TRAVEL_TYPE_GROUP = 4;
}

Reply:
Updated the enum with this.